### PR TITLE
Inform students when they are submitting responses on behalf of the team #4578

### LINKED ISSUE #4578: Inform students when they are submitting responses on behalf of the team
When the giver type is TEAM, the students in a team share the same response, we should let the respondent know that it is a team question on the student's submission page. 

Edit: see discussion in #4576

### PR DESCRIPTION
**Pull request by Sumei1009**
fixes #4578

### CODE REVIEW

**Comment by Sumei1009** [comment]
The warning is showed under the visibility message, displayed as brown color. 
screen shot 2016-03-30 at 22 38 04
screen shot 2016-03-30 at 22 38 15

**Comment by damithc** [comment]
behave --> behalf
The user is not editing a question, submitting responses.

File: src/main/java/teammates/ui/template/FeedbackSubmissionEditQuestion.java [hunk 1]
```diff
@@ -27,6 +29,7 @@ public FeedbackSubmissionEditQuestion(FeedbackQuestionAttributes questionAttribu
         questionText = questionAttributes.getQuestionDetails().questionText;
         visibilityMessages = questionAttributes.getVisibilityMessage();
         questionType = questionAttributes.questionType;
+        giverType = questionAttributes.giverType;
```
Comment (kanghj) [inline src/main/java/teammates/ui/template/FeedbackSubmissionEditQuestion.java]: I don't think this <code>FeedbackSubmissionEditQuestion</code> needs to know the question's giver type. It only needs to know if the message needs to be displayed, so just passing that in is sufficient.

File: src/test/resources/pages/studentFeedbackSubmitPageFullyFilled.html [hunk 2]
```diff
@@ -353,6 +353,9 @@
               The receiving teams can see your response, and your name.
             </li>
           </ul>
+          <p class="text-warning">
+            Please note that you are editing this question on behave of your team.
```
Comment (kanghj) [inline src/test/resources/pages/studentFeedbackSubmitPageFullyFilled.html]: Remember to update the tests

**Comment by thyageshm** [comment]
@Sumei1009 any reason for closing this pr?

**Pull request closed without merging**